core/src/main/java/pl/agh/edu/ui/screen/main/MainScreen.java

```diff
@@ -18,7 +18,7 @@
 
 public class MainScreen implements Screen {
 
-	public final GdxGame game = (GdxGame) Gdx.app.getApplicationListener();
+	public static Engine engine = null;
```

Comment:
może pasuje opakować to w optional ale bym zostawił tak jak jest bo będzie to pewnie zmieniane jeszcze 5 razy

Reply:
ale we wszystkich miejscach dalej używamy engine z game?

Reply:
Poprawiłem tam gdzie mi się pokazało jak odpalałem ale pewnie trzeba będzie na bieżąco poprawiać